### ISSUE TITLE
Labeling Instructions in Markdown format

### ISSUE THREAD

**Issue by DougTrajano** [heartexlabs/label-studio#1339]
It Will be very helpful if the Labeling Instructions feature in `Project > Settings` accepts markdown and parses it to HTML.

I would like to add instructions in markdown format like:

```markdown
## Case A

Bla bla bla bla

## Case B

blu blu blu
```

And have it parsed to the HTML in the PopUp interface.

**Comment by makseq** [comment]
@DougTrajano Thank you for your idea, what do you think about WYSIWYG editor?

**Comment by DougTrajano** [comment]
> @DougTrajano Thank you for your idea, what do you think about WYSIWYG editor?

I think that will be great too. Today I spent some time testing Label Studio and it was amazing, I think that the WYSIWYG editor is the pattern used in the Labeling Interface, isn't it?

Labeling Interface was the feature that I most liked in Label Studio because it is very flexible and very easy to understand and use. Really great work there! 💯

**Comment by makseq** [comment]
> WYSIWYG editor is the pattern used in the Labeling Interface, isn't it?
Yeah, in some old versions we had WYSIWYG. I hope we will get it back soon. 

Thank you very much for your kind words!

**Comment by makseq** [comment]
@krnithishkumar We wanted to add this during 1.4.

**Comment by tilusnet** [comment]
+1 from me, I'd like formatted instructions too.

**Comment by lulia0228** [comment]
I'd like formatted response. But I don`t  how to set up for it.

**Comment by AbubakarSaad** [comment]
Hello,

This feature now exist.
